app/build.gradle

```diff
@@ -141,6 +141,12 @@ android {
             universalApk true
         }
     }
+
+    testOptions {
+        unitTests.all {
+            jvmArgs '-noverify'
```

Comment:
This is necessary because some classes, like GoogleSignInAccount, are obfuscated by Proguard and this cause many problems when mocking.

There is an issue opened for Proguard, and I'm following this issue: https://github.com/mockito/mockito/issues/1138